django_netjsongraph/base/topology.py

```diff
@@ -248,6 +252,19 @@ def update(self, data=None):
                         link.full_clean()
                         link.save()
 
+    def save_snapshot(self):
+        """
+        Saves the snapshot of topology
+        """
+        Snapshot = self.snapshot_model
```

Comment:
you are right, I don't know why I was tricked by my memory. Some code analysis tool compare if you declare a CamelCase variable, but not the default flake8 configuration, ignore this comment